flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/plan/batch/table/stringexpr/SetOperatorsTest.scala

```diff
@@ -22,24 +22,12 @@ import org.apache.flink.api.scala._
 import org.apache.flink.table.api.scala._
 import org.apache.flink.table.planner.utils.TableTestBase
 
-import org.junit.{Ignore, Test}
+import org.junit.Test
 
 import java.sql.Timestamp
 
 class SetOperatorsTest extends TableTestBase {
 
-  @Ignore("Support in subQuery in ExpressionConverter")
-  @Test
-  def testInWithFilter(): Unit = {
```

Comment:
Why this is removed? I run this test and can get a plan. Is the plan incorrect? cc @godfreyhe 

```
HashJoin(joinType=[LeftSemiJoin], where=[=(c, a1)], select=[a, b, c], build=[right], tryDistinctBuildRow=[true])
:- Exchange(distribution=[hash[c]], shuffle_mode=[BATCH])
:  +- TableSourceScan(table=[[default_catalog, default_database, A, source: [TestTableSource(a, b, c)]]], fields=[a, b, c], reuse_id=[1])
+- Exchange(distribution=[hash[a1]])
   +- LocalHashAggregate(groupBy=[a1], select=[a1])
      +- Calc(select=[a AS a1], where=[=(b, _UTF-16LE'two')])
         +- Reused(reference_id=[1])
```

Reply:
Sorry, it seems this is deleted by accident